mapit_labour/templates/mapit_labour/uprn.html

```diff
@@ -11,8 +11,8 @@ <h2>UPRN: {{ uprn.uprn }}</h2>
       <li>Postcode: <a href="{% url "mapit-postcode" postcode=uprn.postcode format="html" %}">{{ uprn.postcode }}</a>
       <li>OSGB E/N: {{ uprn.easting }}, {{ uprn.northing }}
       <li>{% trans "WGS84 lat/lon" %}: <a href="https://tools.wmflabs.org/geohack/geohack.php?params={{ uprn.wgs84_lat|floatformat:-6 }};{{ uprn.wgs84_lon|floatformat:-6 }}">{{ uprn.wgs84_lat|floatformat:-6 }}, {{ uprn.wgs84_lon|floatformat:-6 }}</a>
-      {% url json_view uprn=uprn.uprn as json_url %}
-      <li>{% blocktrans %}Get <a href="{{ json_url }}">this data as JSON</a>{% endblocktrans %}</li>
+      {% url json_view uprn=uprn.uprn format='json' as json_url %}
```

Comment:
Just wondered why you added this (no issue with it)

Reply:
The `format='json'` part? I'm afraid I can't remember :/